demo/source/test.ts

```diff
@@ -274,12 +274,14 @@ it('no trades, sync before/after', async () => {
 })
 
 fit('one trade', async () => {
-	await resetUniswapAndAccount(contracts.uniswapExchange, contracts.token0, contracts.token1, rpcSignerAddress, 1n, 1n)
-	const blockNumber = await rpc.getBlockNumber() // block where liquidity was minted, implicit sync call, price is 1:1 at the end of this block
-	await setPrice(contracts.uniswapExchange, contracts.token0, contracts.token1, 2n, 1n) // one block at 1:1 price, followed by a block with a sync call in it and 2:1 price at end
-	await contracts.uniswapExchange.sync() // block with 2:1 price for full duration and sync included
+	await resetUniswapAndAccount(contracts.uniswapExchange, contracts.token0, contracts.token1, rpcSignerAddress, 5n, 1n)
```

Comment:
Consider: Change this back to setting starting price to 1:1, just because calling `setPrice` burns through our unlimited tokens quickly.  😬  I should really mint far more tokens...

Reply:
you mean in the initial liquidity?

Reply:
The second to last parameter in this line being 5 means that you are seeding with 5x token 0 to token 1 ratio.  Since this test only wants 1:1, might as well start with 1:1 is all.